recipe/meta.yaml

```diff
@@ -1,25 +1,16 @@
-{% set version = "2016.2.28" %}
+{% set version = "2016.8.2" %}
 
 package:
   name: certifi
   version: {{ version }}
 
 source:
   fn: certifi-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/c/certifi/certifi-{{ version }}.tar.gz
-  sha256: 5e8eccf95924658c97b990b50552addb64f55e1e3dfe4880456ac1f287dc79d0
-  patches:
-    # Uses `distutils` if `setuptools` is not found.
-    # This will allow it to be a dependency of `setuptools`.
-    # Has already been accepted upstream, but is not yet released.
-    # The relevant PR is linked below.
-    #
-    # https://github.com/certifi/python-certifi/pull/43
-    #
-    - python-certifi_PR_43.diff
```

Comment:
We should remove the diff file too. Please see PR ( https://github.com/conda-forge/certifi-feedstock/pull/11 ).